calendarsubscription/store.go

```diff
@@ -48,13 +49,13 @@ func NewStore(ctx context.Context, db *sql.DB, apiKeyring keyring.Keyring, oc on
 		authUser: p.P(`
 			UPDATE user_calendar_subscriptions
 			SET last_access = now()
-			WHERE NOT disabled AND id = $1
+			WHERE NOT disabled AND id = $1 AND date_trunc('second', created_at) = $2
 			RETURNING user_id
 		`),
 
 		findOne: p.P(`
 			SELECT
-				id, name, user_id, disabled, schedule_id, config, last_access
+				id, name, user_id, disabled, schedule_id, config, last_access, created_at
```

Comment:
I think this was leftover: `sql: expected 8 destination arguments in Scan, not 7`
```suggestion
				id, name, user_id, disabled, schedule_id, config, last_access
```